modules/tinymce/src/core/main/ts/newline/NewLineUtils.ts

```diff
@@ -1,5 +1,5 @@
 import { Fun, Optional, Unicode } from '@ephox/katamari';
-import { SugarElement } from '@ephox/sugar';
+import { SelectorExists, SugarElement } from '@ephox/sugar';
```

Comment:
```suggestion
import { SugarElement } from '@ephox/sugar';
```
This is why the LGTM warning is occurring.